frontend/src/components/App/index.module.css

```diff
@@ -0,0 +1,28 @@
+.editorPanel > :global.rs-panel-body {
+    height: inherit
+}
+
+.loader {
+    z-index: 10;
+}
+
+.runButton {
+    float: right
+}
+
+@media (min-width: 992px) {
+    .grid {
+        display: flex;
+    }
+    .editorPanel {
+        /** (Editor bottom padding - Panel header - Navbar) */
+        min-height: calc(100vh - 10px - 60px - 56px);
```

Comment:
Ich glaube das wird mobile wegen der 100vh nicht gut aussehen.

https://chanind.github.io/javascript/2019/09/28/avoid-100vh-on-mobile-web.html

Reply:
(Und die Seite so aufzubauen ist schon ein wenig hacky 🗡 )

Reply:
it's no mobile first application, so I guess it's fine. o.O